modin/pandas/dataframe.py

```diff
@@ -4,7 +4,7 @@
 
 import pandas
 from pandas.api.types import is_scalar
-from pandas.compat import to_str, string_types, cPickle as pkl
+from pandas.compat import to_str, string_types, numpy, cPickle as pkl
```

Comment:
can we alias `numpy as numpy_compat` or something like that? It will be confusing if we just use `numpy`.